src/harness.ts

```diff
@@ -500,9 +365,9 @@ export class Harness<W extends WidgetBase<WidgetProperties>> extends Evented {
 	public sendEvent<I extends EventInit>(type: string, options: HarnessSendEventOptions<I> = {}): this {
 		let { target = this.getDom(), key, ...sendOptions } = options;
 		if (key) {
-			const vnode = findVNodebyKey(this._currentRender, key);
-			if (vnode && vnode.domNode) {
-				target = vnode.domNode as Element;
+			const dnode = findDNodeByKey(this._widgetHarness.lastRender, key);
+			if (dnode && !isWNode(dnode)) {
```

Comment:
any reason why not `isHNode` instead of `!isWNode`?

Reply:
Not that I can think of... Wasn't thinking.